lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -12,11 +12,9 @@ export default class ManagedCdp {
 
     assert(ilk && typeof ilk === 'string', 'Must specify ilk');
     this.ilk = ilk;
-
     this._cdpManager = cdpManager;
-    this.currency = cdpManager
-      .get(ServiceRoles.CDP_TYPE)
-      .getCdpType(null, ilk).currency;
+    this._cdpType = cdpManager.get(ServiceRoles.CDP_TYPE).getCdpType(null, ilk);
+    this.currency = this._cdpType.currency;
   }
 
   async getCollateralValue() {
```

Comment:
should `getCollateralValue` be updated to allow for specifying if you want the return value to be in USD or GEM units, as is done in the SCD version? Then a few of the methods below could be updated to not multiply the result by the collateral price.  Because as is, I think its a little confusing that this is called getCollateralValue, as opposed to something like getCollateralAmount, since it returns the number of collateral units, not the usd value of the collateral.

Reply:
yeah I think we should have amount and value and keep the separate value is the amount in USD and amount is the value in gem unites